JM/CF/Scripts/3_Game/CommunityFramework/Timer/CF_DelayCall.c

```diff
@@ -0,0 +1,69 @@
+class CF_DelayCall : CF_TimerBase
+{
+	protected Managed m_Instance;
+	protected string m_Function;
+	protected ref CF_TimerParam m_Params;
+
+	protected int m_Delay;
+	
+	void CF_DelayCall()
+	{
+	}
+
+	/**
+	 * @param instance The instance that owns the function
+	 * @param function The function that is called
+	 * @param interval The time between each function call, in milliseconds.
+	 * @param params The static parameters that are passed into the function
+	 * 
+	 * @note The function passed must have a signature that starts with 'CF_TimerBase,float` and then continues with the matching 'params'.
+	 */
+	static CF_TimerBase Create(Managed instance, string function, int delay, CF_TimerParam params = null)
```

Comment:
> Does it need managed here? Same discussion we had about normal timers - if we are responsible to handle the case when the instance that we should invoke becomes invalid.
Changing it to class would allow this to be used with existing custom classes that use inheritance.

My mistake in forgetting about `CF_DelayedCall`

> Change the delay param to be float to fit the rest of the timer functions.

Safer to use an int, since `CF_DelayedCall` is designed to operate over a very long time. `CF_Timer` uses a float because the interval between each call is expected to be within a safe range for floating-point errors it is fine. `CF_DelayedCall` is expected to only call the function once over minutes or hours later, this can have floating-point errors if a modder wanted a precise time.